packages/blade/src/components/Box/BaseBox/baseBoxStyles.ts

```diff
@@ -107,6 +107,24 @@ export const getElevationValue = (
       getIn(theme, `elevation.${responsiveElevationValue!}`);
 };
 
+const getBorderStyleValue = (
+  borderStyle: BaseBoxProps['borderStyle'],
+  breakpoint?: keyof Breakpoints,
+  hasBorder?: boolean,
+  // Using any as return type because borderStyle's type is incompatible with borderBottomStyle. There are ways to fix it but anyway since its internal function. Taking an easy way out
```

Comment:
Can't we add a `BorderBottomStyleType | BorderStyleType` and then enforce one or the other where it's being consumed? `any` has a way of hiding and creeping in later on at unexpected places 🙈

Reply:
done